realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -411,6 +411,7 @@ private static void initializeSyncedRealm(Realm realm) {
                 realm.setVersion(newVersion);
                 commitChanges = true;
             }
+            schema.close();
```

Comment:
Shouldn't the schema lifecycle follow the Realm instance lifecycle instead? E.g. release it when we release the last instance?